optimizely/event/event_processor.py

```diff
@@ -195,6 +200,12 @@ def _flush_queue(self):
 
     log_event = EventFactory.create_log_event(to_process_batch, self.logger)
 
+    if self.notification_center is not None:
```

Comment:
Should we be validating that `notification_center` is an instance of `notification_center.NotificationCenter` or does that happen elsewhere?